src/algorithms/policy_gradient/multi_thread_env.jl

```diff
@@ -126,6 +131,7 @@ function RLBase.is_terminated(env::MultiThreadEnv)
 end
 
 function RLBase.legal_action_space_mask(env::MultiThreadEnv)
+    N = ndims(env.states)
```

Comment:
Assume this is supposed to be here, does not really have to do with this PR but seemed reasonable to add.

Reply:
Thanks, we can test and improve it later.